packages/core/src/analytics/integrations/Omnitracking/definitions.js

```diff
@@ -523,6 +524,83 @@ export const trackEventsMapper = {
     wishlistId: data.properties?.wishlistId,
     lineItems: getProductLineItems(data),
   }),
+  [eventTypes.PRODUCT_UPDATED]: data => {
+    const eventList = [];
+    const properties = data.properties;
+
+    if (properties.colour && properties.oldColour !== properties.colour) {
+      // color changed event
+      const additionalParameters = {};
+      if (properties?.oldColourId && properties?.colourId) {
+        additionalParameters[
+          'colourList'
+        ] = `${properties?.oldColourId}, ${properties?.colourId}`;
+      } else {
+        logger.warn(
+          `[Omnitracking] - Event ${data.event} property oldColourId and colourId can be used on track.
+                        It was tracked an change color event, and in this event these parameters are relevant to track.`,
+        );
+      }
+
+      eventList.push({
+        ...additionalParameters,
+        tid: 2098,
+        productId: properties?.id,
+        actionArea: properties?.from,
+      });
+    }
+
+    if (properties.size && properties.oldSize !== properties.size) {
+      // size changed event
+
+      const additionalParameters = {};
+      if (properties?.oldSizeScaleId && properties?.sizeScaleId) {
+        additionalParameters[
+          'scaleList'
+        ] = `${properties?.oldSizeScaleId}, ${properties?.sizeScaleId}`;
+      } else {
+        logger.warn(
+          `[Omnitracking] - Event ${data.event} property oldSizeScaleId and scaleId can be used on track.
```

Comment:
similar to https://github.com/Farfetch/blackout/pull/497/files#r975218755

Reply:
Updated!